frontend/tests/utilsv2/core/hot-reloader.ts

```
@@ -0,0 +1,413 @@
+		if (bundledCode === this._currentCode) {
+			return;
+		}
+		this._currentCode = bundledCode;
+
+		const finalCode = ts.transpileModule(bundledCode, {
+			compilerOptions: { module: ts.ModuleKind.CommonJS }
+		}).outputText;
+
+		try {
+			// entryPoint is the _start function as defined in the main.ts of the Hot Reloader path.
+			const entryPoint: (args: any) => Promise<void> = eval(finalCode);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)